Timeline.cpp

```diff
@@ -106,7 +127,7 @@ void TweenDuino::Timeline::update(unsigned long newTime) {
     // Maintenance Note: Very similar looping logic in TweenDuino::Timeline::add & restart.
     // If you change this line here, you might need to change it there.
     for (int i = 0; i < TWEEN_TIMELINE_SIZE && tweens[i].tween != nullptr; i++) {
-        TimelineEntry entry = tweens[i];
+		TweenDuino::Timeline::TimelineEntry &entry = tweens[i];
```

Comment:
Appreciate the use of references.  I think there are probably other cases where I'm accidentally causing copies of tweens to happen.

If you could, please replace the tab indents with space indents.